src/antq/core.clj

```diff
@@ -151,13 +181,16 @@
         (log/info))))
 
 (defn- assoc-latest-version
-  [dep options]
+  [dep options forced-artifacts]
```

Comment:
It's hard to read because you're shadowing the function name, so could you use a name different from the function name?